tests/capi/doc/chfl_atom/list_properties.c

```diff
@@ -0,0 +1,31 @@
+// Chemfiles, a modern library for chemistry file reading and writing
+// Copyright (C) Guillaume Fraux and contributors -- BSD license
+
+#include <chemfiles.h>
+#include <assert.h>
+#include <stdlib.h>
+
+int main() {
+    // [example]
+    CHFL_ATOM* atom = chfl_atom("Na");
+    CHFL_PROPERTY* property = chfl_property_double(-23);
+
+    chfl_atom_set_property(atom, "this", property);
+    chfl_atom_set_property(atom, "that", property);
+    chfl_property_free(property);
+
+    uint64_t count = 0;
+    chfl_atom_properties_count(atom, &count);
+    assert(count == 2);
+
+    const char* names[2] = {NULL};
+    chfl_atom_list_properties(atom, names, count);
+
+    // Properties are not ordered
```

Comment:
I'm guessing this is implementation dependent due to an unordered map.

Reply:
Yes. I think it it better to document it as not ordered for now, this can be changed later without breaking code

Reply:
This is fine, I just wanted to make sure.